<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputBindingDialog</class>
 <widget class="QDialog" name="InputBindingDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Bindings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Bindings for Controller0/ButtonCircle</string>
     </property>
     <property name="buddy">
      <cstring>bindingList</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="bindingList"/>
   </item>
   <item>
    <widget class="QWidget" name="sensitivityWidget" native="true">
     <layout class="QGridLayout" name="sensitivityLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="sensitivityLabel">
        <property name="text">
         <string>Sensitivity:</string>
        </property>
        <property name="buddy">
         <cstring>sensitivity</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="sensitivity">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TickPosition::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="sensitivityValue">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="deadzoneLabel">
        <property name="text">
         <string>Deadzone:</string>
        </property>
        <property name="buddy">
         <cstring>deadzone</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="deadzone">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TickPosition::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="deadzoneValue">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addBinding">
       <property name="text">
        <string>Add Binding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBinding">
       <property name="text">
        <string>Remove Binding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBindings">
       <property name="text">
        <string>Clear Bindings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bindingList</tabstop>
  <tabstop>sensitivity</tabstop>
  <tabstop>deadzone</tabstop>
  <tabstop>addBinding</tabstop>
  <tabstop>removeBinding</tabstop>
  <tabstop>clearBindings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
